Jobs/GenerateTag.yml

```diff
@@ -0,0 +1,76 @@
+## @file
+# Template file used to generate tags on ADO. This template requires that the
+# consumer specifies this repository as a resource named mu_devops.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+  - name: major_version
+    displayName: The major version.
+    type: string
+    default: ""
+  - name: git_name
+    displayName: Name to use for creating tag.
+    type: string
+    default: ""
+  - name: git_email
+    displayName: Email to use for creating tag.
+    type: string
+    default: ""
+  - name: notes_file
+    displayName: Path to the notes file to generate.
+    type: string
+    default: "ReleaseNotes.md"
+  - name: extra_prepare_steps
+    displayName: Extra Prepare Steps
+    type: stepList
+    default:
+      - script: echo No extra prepare steps provided
+
+jobs:
+  - job: Create_Release_Tag
+    steps:
+      - checkout: self
+        clean: true
+        fetchTags: true
+        persistCredentials: true
+        path: "target"
+
+      - checkout: mu_devops
+        path: "mu_devops"
+        fetchDepth: 0

```

Comment:
I think fetchepth can be omitted or set to 1 to reduce the amount of data done in the checkout. We only need the latest mu_devops.